src/app/core/entity/entity-actions/entity-actions.service.ts

```diff
@@ -140,24 +186,28 @@ export class EntityActionsService {
     const progressDialogRef = this.confirmationDialog.showProgressDialog(
       $localize`:Entity action progress dialog:Processing ...`,
     );
-    const result = await this.entityAnonymize.anonymizeEntity(entity);
+    let result = new CascadingActionResult();
+    for (let entity of entities) {
+      result.mergeResults(await this.entityAnonymize.anonymizeEntity(entity));
+    }
     progressDialogRef.close();
 
     if (result.potentiallyRetainingPII.length > 0) {
       await this.confirmationDialog.getConfirmation(
         $localize`:post-anonymize related PII warning title:Related records may still contain personal data`,
-        $localize`:post-anonymize related PII warning dialog:Some related records (e.g. notes) may still contain personal data in their text. We have automatically anonymized all records that are linked to ONLY this ${
-          entity.getConstructor().label
-        }.
+        $localize`:post-anonymize related PII warning dialog:Some related records (e.g. notes) may still contain personal data in their text. We have automatically anonymized all records that are linked to ONLY ${textForAnonymizeEntity}.
```

Comment:
this doesn't work here, because the sentence structure is different (for delete it was the same structure).

Reply:
I fixed that.